README.md

```diff
@@ -261,73 +259,350 @@ The SDK provides a simple interface for all the [media APIs mentioned here](http
+createFolderRequest.FolderName="test1";
+createFolderRequest.ParentFolderPath="/";
+ResponseMetaData resultEmptyBlock = ImageKit.createFolder(createFolderRequest);
+
+```
+
+**17. Delete Folder**
+
+Accepts an object of class `DeleteFolderRequest` specifying the parameters to be used to delete folder. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/delete-folder) can be passed via their setter functions to get the results.
+
+```java
+DeleteFolderRequest deleteFolderRequest = new DeleteFolderRequest();
+deleteFolderRequest.FolderPath="/test1";
+ResultNoContent resultNoContent = ImageKit.deleteFolder(deleteFolderRequest);
+
+```
+
+**18. Copy Folder**
+
+Accepts an object of class `CopyFolderRequest` specifying the parameters to be used to copy folder. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/copy-folder) can be passed via their setter functions to get the results.
+
+```java
+CopyFolderRequest copyFolderRequest = new CopyFolderRequest();
+copyFolderRequest.SourceFolderPath="/Gallery/test";
+copyFolderRequest.DestinationPath="/";
+ResponseMetaData resultOfFolderActions = ImageKit.copyFolder(copyFolderRequest);
+
+```
+
+**19. Move Folder**
+
+Accepts an object of class `MoveFolderRequest` specifying the parameters to be used to move folder. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/move-folder) can be passed via their setter functions to get the results.
+
+```java
+MoveFolderRequest moveFolderRequest = new MoveFolderRequest();
+moveFolderRequest.SourceFolderPath="/Gallery/test";
+moveFolderRequest.DestinationPath="/";
+ResponseMetaData resultOfFolderActions = ImageKit.moveFolder(moveFolderRequest);
+
+```
+
+**20. Get Bulk Job Status**
+
+Accepts the jobId to get bulk job status as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/copy-move-folder-status).
+
+```java
+String jobId = "629f44ac7eb0fe8173622d4b";
+ResponseMetaData resultBulkJobStatus = ImageKit.getBulkJobStatus(jobId);
+
+```
+
+**21. Purge Cache**
+
+Accepts a full URL of the file for which the cache has to be cleared as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/purge-cache).
+
+```java
+ResponseMetaData result=ImageKit.purgeCache("https://ik.imagekit.io/imagekit-id/default-image.jpg");
+
+```
+
+**22. Purge Cache Status**
+
+Accepts a request ID and fetch purge cache status as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/purge-cache-status)
+
+```java
+String requestId="cache-requestId";
+ResponseMetaData result=ImageKit.getPurgeCacheStatus(requestId);
+
+```
+
+**23. Get File Metadata**
+
+Accepts the file ID and fetches the metadata as per the [API documentation here](https://docs.imagekit.io/api-reference/metadata-api/get-image-metadata-for-uploaded-media-files)
+
+```java
```

Comment:
Use `.NET` and not `java`. Please be very mindful of these kinds of copy-paste mistakes.

Reply:
@nitanandsinghupw `.net` doesn't seem to be correct as well. Syntax highlighting is not working as can be [seen here](https://github.com/nitanandsinghupw/imagekit-dotnet/tree/dev).